scss/tech.scss

```diff
@@ -0,0 +1,60 @@
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+
+.boxes {
+  display: flex;
+  flex-flow: row;
+  flex-wrap: wrap;
+  padding: 0px 100px;
+}
+
+.box {
+  background: $green;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  min-height: 310px;
+  margin: 25px;
+  position: relative;
+  text-align: center;
+  width: 200px;
+  &::before {
+    border: 1px solid $green;
+    border-radius: 8px;
+    right: 10px;
+    bottom: 10px;
+    content: '';
+    height: 100%;
+    position: absolute;
+    width: 100%;
+    z-index: -1;
+    }
+}
+
+.box-title {
+  font-size: $text-medium;
+  margin-bottom: 10px;
+}
+
+.box-release {
+  font-size: $text-small;
+  font-weight: bold;
+  margin-bottom: 10px;
+}
+
+.box-description {
+  font-size: $text-small;
+  margin-top: 30px;
+}
+
+.box-cover {
+  display: block;
+  margin: 10px 0 7px 50px;
```

Comment:
hmmmm this looks a bit weird, the 10px should be as padding in the parent and the 50px shouldn't be needed in you add `align-items: center` in the parent to center each child horizontally